Components/Avatar/avatar.html

```diff
@@ -65,6 +65,9 @@
             <li class="component-links">
                 <a href="../Utilities/utilities.html">Utilities</a>
             </li>
+            <li class="component-links">
+                <a href="../Grid//grid.html">Grid</a>
```

Comment:
// before grid.html change it to /grid.html , here and at all other places.

Reply:
corrected